content/5_Plat/Bitsets.mdx

```diff
@@ -73,7 +73,52 @@ void init() {
 		if (D.get(i) == i) comps.push_back(D.size(i));
 }
 ```
-
+```py
```

Comment:
use `<LanguageSection>` as described in https://usaco.guide/general/adding-solution?lang=cpp

Reply:
Done. Btw thank you for being so patient with me :))

Reply:
you'll also need to use CPPSection and PySection within the LanguageSection to surround the code for the respective languages. make sure to check that the output renders correctly when using the editor: https://usaco.guide/editor

Reply:
Done, It renders perfectly.

Reply:
not quite, but I fixed it